app/controllers/api/v1/lobbies_controller.rb

```diff
@@ -3,19 +3,41 @@
 module Api
   module V1
     class LobbiesController < AuthenticatedController
-      before_action :require_authentication, except: [:from_code, :join]
+      before_action :require_authentication, except: [:from_code, :join, :answer]
+      before_action :set_lobby, only: [:join, :start, :answer, :show, :update, :destroy]
       before_action :require_authorisation, only: [:show, :update, :destroy]
       before_action :set_quiz, only: [:index, :create]
 
       def join
-        lobby = Lobby.find(params[:id])
+        @player = @lobby.players.create(player_params)
 
-        @player = lobby.players.create(player_params)
-
-        Pusher.trigger(lobby.code, Lobby::PLAYER_JOIN, { id: @player.id })
+        Pusher.trigger(@lobby.code, Lobby::PLAYER_JOIN, { id: @player.id })
         render :player, status: :created
       end
 
+      def start
+        @lobby.status = :in_progress
+        @lobby.save
+
+        Pusher.trigger(@lobby.code, Lobby::LOBBY_START)
+        render :show
+      end
+
+      def answer
+        @player = Player.find_by(id: params[:player_id])
+
+        return head :not_found if @player.nil?
+
+        @answers = Question.find_by!(quiz_id: @lobby.quiz.id, order: @lobby.current_question_index).answers
+        PlayerAnswer.where(answer: @answers, player: @player).delete_all
```

Comment:
Use `destroy_all` instead of `delete_all` as it will also run any callbacks that are on the model. `delete_all` simply executes a `DELETE` statement without taking into account anything else defined on the model.